frontend/src/components/app/app.tsx

```diff
@@ -135,6 +136,23 @@ const App: FC = () => {
             </AuthorizedWrapper>
           }
         />
+        <Route
+          path={AppRoute.INTERVIEWS_$ID}
+          element={
+            <AuthorizedProtectedRoute
+              permissions={[PermissionKey.MANAGE_INTERVIEWS]}
+              component={<Interview />}
+            />
+          }
+        />
```

Comment:
Why do we need this? we have this route above

Reply:
Oh, strange one 😕